test/helper.rb

```diff
@@ -16,40 +18,9 @@
 require_relative 'models'
 
 def assert_queries(num = 1)
-  ActiveRecord::SQLCounter.log = []
-  yield
-ensure
-  assert_equal num, ActiveRecord::SQLCounter.log.size, "#{ActiveRecord::SQLCounter.log.size} instead of #{num} queries were executed.#{ActiveRecord::SQLCounter.log.size == 0 ? '' : "\nQueries:\n#{ActiveRecord::SQLCounter.log.join("\n")}"}"
```

Comment:
never do assertions in an ensure: when something blows up, the assertion hides it by blowing up itself :)